mac/macdeploy.sh

```diff
@@ -19,23 +20,50 @@ cd bin
 # copy libQupZilla into bundle
 cp $LIBRARY_NAME QupZilla.app/Contents/MacOS/
 
-# copy all plugins into bundle
-test -d QupZilla.app/Contents/Resources/plugins || mkdir QupZilla.app/Contents/Resources/plugins
-cp plugins/*.dylib QupZilla.app/Contents/Resources/plugins/
+# copy all QupZilla plugins into bundle
+test -d $PLUGINS || mkdir $PLUGINS
+cp plugins/*.dylib $PLUGINS/
 
 # fix libQupZilla
 install_name_tool -change $LIBRARY_NAME @executable_path/$LIBRARY_NAME QupZilla.app/Contents/MacOS/QupZilla
 
 # fix plugins
-for plugin in QupZilla.app/Contents/Resources/plugins/*.dylib
+for plugin in $PLUGINS/*.dylib
```

Comment:
Well that wouldn't be me then... all I did was abstract it... but according to the Qt docs custom Libraries should be set so they "announce themselves" for use.